src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -149,7 +168,7 @@ HttpStatusCode mongoSubscribeContext
                                              tenant,
                                              xauthToken,
                                              servicePathV,
-                                             requestP->expression.q);
+                                             &restriction);
```

Comment:
After looking this, I guess that the `const Restriction& rest` alternative is possible.